starter-code/src/clue.js

```diff
@@ -82,13 +123,98 @@ name: Observatory
 name: Theater
 name: Guest House
 name: Patio
-*/
+ */
+
+
+const DiningRoom = {
+    name: "Dining Room"
+};
```

Comment:
only class-variables are written Uppercased, variables like this one should start lowerCased-> const diningRoom
..not breaking your code here, but looks confusing cause someone might expect a class:)